src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -454,7 +486,11 @@ public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verif
             }
             else if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX)
             {
-                return false;
+                // Please note: Linux/MAC case sensitive for env var name.
+                string signVerifyEnvVariable = _environmentVariableReader.GetEnvironmentVariable("DOTNET_OPT_IN_SECURE_PACKAGE_VERIFICATION");
+
+                // Not opt-out option, only opt-in feature.
+                return !string.IsNullOrEmpty(signVerifyEnvVariable);
```

Comment:
you still need to check for the value being `true` right?

Reply:
No, value doesn't matter since we doesn't support opt-out using value.
https://github.com/NuGet/Client.Engineering/issues/882#issuecomment-815254218

Reply:
I feel like we're making it needlessly complicated. 
I've rarely seen env vars where the value doesn't matter at all. 

MSBuild conditions are a great example of how to treat strings as bools. You check for `true` only. Everything else is default, whatever that may be.

Reply:
Also, if you read the documentation for GetEnvironmentVariable, `` means that the env var is set to empty. 
`null` means the env var is not set.

Reply:
@nkolev92 
Can you comment here? https://github.com/NuGet/Client.Engineering/issues/882#issuecomment-815254218

Reply:
yeah, I did just a 1 minute before you asked me to :D

Reply:
Please review again.